tests/io/test_cif.py

```diff
@@ -0,0 +1,147 @@
+import filecmp
+import tempfile
+from os.path import exists
+
+import gemmi
+import pytest
+from pandas.testing import assert_frame_equal
+
+import reciprocalspaceship as rs
+from reciprocalspaceship.utils import in_asu
+
+
+def test_read_merged(IOtest_cif):
+    """Test rs.read_cif() with merged SF CIF file"""
+    dataset = rs.read_cif(IOtest_cif)
+
+    assert dataset.spacegroup.number == 146
+    assert dataset.columns.to_list() == ["FreeR_flag", "FC", "PHIC"]
```

Comment:
This line causes the test to fail because there are two additional columns (IMEAN and SIGIMEAN). I'm guessing this is expected, and that the test should be updated to reflect the contents of `1CTJ-sf.cif`.

Reply:
I'm not sure why this is occuring, the `1CTJ-sf.cif` file does not contain additional columns.

Reply:
Looking at the cif, the file has columns with the following names:
```
loop_
_refln.crystal_id
_refln.wavelength_id
_refln.scale_group_code
_refln.index_h
_refln.index_k
_refln.index_l
_refln.status
_refln.F_calc
_refln.F_squared_meas
_refln.F_squared_sigma
_refln.phase_calc
```
I'm guessing that "F_squared_meas" and "F_squared_sigma" get turned into IMEAN and SIGIMEAN by `gemmi`. At the very least, the tests fail because those two columns are present but the test doesn't expect them.

Reply:
Ahh I see. I ran the following locally
```python
cif = 'tests/data/fmodel/1CTJ-sf.cif'
dataset = rs.read_cif(cif)
dataset.columns.to_list()
```
and it returns ['FreeR_flag', 'FC', 'PHIC'].

Reply:
ok, I see the trouble. Your clone of this repo is a few months out of date, so the version you are testing with uses `gemmi==0.6.0`. If you update to `gemmi==0.6.2` (which is what the current setup.py uses), you'll get the IMEAN and SIGIMEAN columns that are causing the issue. 

When the tests are run here, it only incorporates the files you change, so they are effectively being run on top of the latest gemmi. If you change your local copy to use `gemmi==0.6.2`, you'll be able to test locally in a way that is consistent with the tests through GitHub Actions.

Reply:
Okay, I updated gemmi to version `0.6.2` and now the function returns ["FreeR_flag", "IMEAN","SIGIMEAN","FC", "PHIC"]. 

I updated test_cif.py to reflect these changes, it should pass the tests now.